### DISCUSSION TITLE
Does webpack have an api for custom dependency analysis?

### DISCUSSION THREAD

**Discussion by kdy1** [webpack/webpack#13408] [Q&A]
Hi.
I'm currently looking for a way to speed up webpack and I think using the parser of [swc][] can improve performance.
Does webpack provide an api to replace it?

If not, will you guys accept a PR for it?

[swc]: https://swc.rs

**Comment by alexander-akait** [comment]
https://github.com/swc-project/swc-loader

**Reply by kdy1** [reply]
I mean a way to skip dependency analysis of webpack.
(I'm author of the swc)

Can I return dependency graph from the loader?

**Reply by alexander-akait** [reply]
> I mean a way to skip dependency analysis of webpack.

No, `dependency graph` is internal structures, but I will be glad to hear ideas

**Reply by alexander-akait** [reply]
Maybe you can open an issue where we continue discussions, I think we will found a way
